Applications/Utils/ModelPreparation/PartitionMesh/NodeWiseMeshPartitioner.cpp

```diff
@@ -329,24 +364,28 @@ NodeWiseMeshPartitioner::getNumberOfIntegerVariablesOfElements(
     return nmb_element_idxs;
 }
 
-void NodeWiseMeshPartitioner::writePropertiesBinary(
+void NodeWiseMeshPartitioner::writeNodePropertiesBinary(
     const std::string& file_name_base) const
 {
-    auto const& property_names(_partitioned_properties.getPropertyVectorNames());
+    auto const& property_names(_partitioned_properties.getPropertyVectorNames(
+        MeshLib::MeshItemType::Node));
     if (property_names.empty())
         return;
+
+    std::size_t const number_of_properties(property_names.size());
+
     const std::string fname_cfg = file_name_base +
-                                  "_partitioned_properties_cfg" +
+                                  "_partitioned_node_properties_cfg" +
                                   std::to_string(_npartitions) + ".bin";
     std::ofstream out(fname_cfg.c_str(), std::ios::binary | std::ios::out);
 
     const std::string fname_val = file_name_base +
-                                  "_partitioned_properties_val" +
+                                  "_partitioned_node_properties_val" +
                                   std::to_string(_npartitions) + ".bin";
     std::ofstream out_val(fname_val.c_str(), std::ios::binary | std::ios::out);
 
-    std::size_t number_of_properties(property_names.size());
-    out.write(reinterpret_cast<char*>(&number_of_properties),
+    out.write(reinterpret_cast<char*>(
+                  const_cast<std::size_t*>(&number_of_properties)),
```

Comment:
The const_cast is necessary since `number_of_properties` is const. But your suggested solution compiles, too. Because the suggested solution is smaller I changed the lines again.